deploy/cloudformation/iiif-service-pipeline.yml

```diff
@@ -518,3 +517,65 @@ Resources:
               Configuration:
                 ProjectName: !Ref PostDeployGitHub
               RunOrder: 1
+
+  DebugPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DebugRoleName
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowReadBuckets
+            Effect: Allow
+            Action:
+              - 's3:GetObject'
+              - 's3:ListBucket'
+            Resource:
+              - !GetAtt CodeS3Bucket.Arn
+              - !Sub ${CodeS3Bucket.Arn}/*
+
+  DeploymentPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DebugRoleName
```

Comment:
I would be more comfortable with this role not able to run the possibly disruptive actions, such as `StartPipelineExecution`, as it could create confusion as to where permissions are being inherited from.